Tests/TABResourceLoaderTests/Mocks/MockURLSession.swift

```diff
@@ -9,13 +9,19 @@
 import Foundation
 @testable import TABResourceLoader
 
+typealias URLSessionCompletionHandler = (Data?, URLResponse?, Error?) -> Void
```

Comment:
I'd personally define this inside the `MockURLSession` as that's where it's being used, unless you feel like it would be used elsewhere too? Just helps keep the global space clean.